app/qml/form/MMFormPage.qml

```diff
@@ -240,8 +240,14 @@ Page {
         font: __style.h3
```

Comment:
Can you please change the `Text` component to `MMText`? It will set the correct line height then. Thanks! :)